lib/param_parsing/parameters.go

```diff
@@ -56,23 +69,67 @@ func GetParameters() Params {
 	logger = lib.InitLogger(params.LogLevel)
 	var err error
 	// Read configuration files
-	if tomlFileExists(params) {
-		params, err = getParamsTOML(params)
-	} else if tfSwitchFileExists(params) {
+	// TOML from Homedir
+	if tomlFileExists(lib.GetHomeDirectory()) {
+		params, err = getParamsTOML(params, lib.GetHomeDirectory())
+		if err != nil {
+			logger.Fatalf("Failed to obtain settings from TOML config in home directory: %v", err)
+		}
+	}
+
+	// TOML from ChDirPath
+	if tomlFileExists(params.ChDirPath) {
```

Comment:
I believe this is also your work at:

https://github.com/warrensbox/terraform-switcher/pull/429

For now, we can exclude: `TOML from ChDirPath` .
See: https://github.com/warrensbox/terraform-switcher/discussions/420#discussioncomment-9624774

Reply:
Yep! :) Because of the way the product selection was playing with the GetParameters (and I assumed the TOML file changes would most likely beat these ones), I merged that branch into here for simplicity